doc/usage.rst

```diff
@@ -139,7 +139,17 @@ few command line options:
    because the default ``pip`` of your system might not support them
    (since version 1.1).
 
-   *Note:* This can produce non-repeatable builds.
+   *Note:* This can produce non-repeatable builds. See also :option:`--upgrade-pip-to`.
+
+.. option:: --upgrade-pip-to <VERSION>
```

Comment:
Should we mark `--upgrade-pip` deprecated and nudge people towards to this, what do you think?

Reply:
Only in 1.4+, let people try the new one first.

Reply:
Good point  👍